src/heat-map/heat-map.component.ts

```diff
@@ -211,14 +218,14 @@ export class HeatMapComponent extends BaseChartComponent {
   }
 
   setColors(): void {
-    this.colors = new ColorHelper(this.scheme, 'linear', this.valueDomain);
+    this.colors = new ColorHelper(this.scheme, this.scaleType, this.valueDomain);
   }
 
   getLegendOptions() {
     return {
-      scaleType: 'linear',
+      scaleType: this.scaleType,
```

Comment:
scaleType here refers to the value scale that the color applies to, which is always linear.

Reply:
Please, I need an example of how to implement the heat chart as seen in the example. I can not see the series grouped by month